jenkins/spark-premerge-build.sh

```diff
@@ -189,65 +188,54 @@ ci_scala213() {
             $MVN_CMD -B $MVN_URM_MIRROR -Dbuildver=$version test -rf tests $MVN_BUILD_ARGS -Dpytest.TEST_TAGS='' \
             -DwildcardSuites=org.apache.spark.sql.rapids.filecache.FileCacheIntegrationSuite
     done
+
     $MVN_CMD -U -B $MVN_URM_MIRROR clean package $MVN_BUILD_ARGS -DskipTests=true
-    cd ..
+    cd .. ## Run integration tests in the project root dir to levarage test cases and resources files
     export TEST_TAGS="not premerge_ci_1"
     export TEST_TYPE="pre-commit"
     export TEST_PARALLEL=5
     # SPARK_HOME (and related) must be set to a Spark built with Scala 2.13
-    SPARK_HOME=$SPARK_HOME PATH=$PATH PYTHONPATH=$PYTHONPATH \
-        ./integration_tests/run_pyspark_from_build.sh
+    #######  SPARK_HOME=$SPARK_HOME PYTHONPATH=$PYTHONPATH \
+    #######    ./integration_tests/run_pyspark_from_build.sh
     # enable avro test separately
-    SPARK_HOME=$SPARK_HOME PATH=$PATH PYTHONPATH=$PYTHONPATH \
+    SPARK_HOME=$SPARK_HOME PYTHONPATH=$PYTHONPATH \
         INCLUDE_SPARK_AVRO_JAR=true TEST='avro_test.py' ./integration_tests/run_pyspark_from_build.sh
     # export 'LC_ALL' to set locale with UTF-8 so regular expressions are enabled
-    SPARK_HOME=$SPARK_HOME PATH=$PATH PYTHONPATH=$PYTHONPATH \
+    SPARK_HOME=$SPARK_HOME PYTHONPATH=$PYTHONPATH \
         LC_ALL="en_US.UTF-8" TEST="regexp_test.py" ./integration_tests/run_pyspark_from_build.sh
-
```

Comment:
Need only to run Unit Tests against 332,340 (SPARK_SHIM_VERSIONS_PREMERGE_SCALA213

Not need to run Unit Tests against 330,340 (SPARK_SHIM_VERSIONS_PREMERGE_UT_2